lang/java/reef-io/src/main/java/org/apache/reef/io/data/loading/api/DataLoadingRequestBuilder.java

```diff
@@ -188,8 +199,50 @@ public DataLoadingRequestBuilder setInputFormatClass(
     return this;
   }
 
+  /**
+   * Allows to specify whether the user wants to use the single data center approach of
+   * loading the data into evaluators {@link SingleDataCenterEvaluatorToPartitionStrategy},
+   * or based on their specified locations in a multi data center topology
+   * {@link MultiDataCenterEvaluatorToPartitionStrategy}.
+   *
+   * @param singleDataCenterStrategy
+   *          true if wants to use the single data center strategy, false otherwise
+   * @return this
+   */
+  public DataLoadingRequestBuilder setSingleDataCenterStrategy(final boolean singleDataCenterStrategy) {
```

Comment:
This doesn't need a setter if it is set automatically based on the other method calls, right?